src/config/eleventy-base-config.js

```diff
@@ -903,7 +903,11 @@ module.exports = function(eleventyConfig) {
   });
 
   eleventyConfig.addFilter('formatPhone', function(phoneStr) {
-    return phoneStr.replaceAll('(', '').replaceAll(') ', '-');
+    phoneStr = phoneStr.replace(/\D/g, '');
```

Comment:
Since this line blows away anything that's not a number regardless of string content, I suggest returning the unmodified input string if `length < 10`.

Reply:
I've added another line with this functionality.

Reply:
Ah, sorry I meant for you to check the length after you remove all non-digits. (similar to your check for `length > 10` below)

As it is now, `formatPhone('hello world')` will return an empty string.  While it could be argued that this is correct since `'hello world'` is not a phone number, I think it's a more graceful fallback to return `'hello world'` instead (i.e. "no phone numbers to format, so here you go: the string unchanged")

It will be especially confusing if `formatPhone('hello')` gives `'hello'` while `formatPhone('hello world')` gives `''` which is how it would work now.

Reply:
I think it's fixed now. `formatPhone` now returns the original string if its length with non-numbers removed is less than 10.

Reply:
Thank you!